less/home-page.less

```diff
@@ -15,16 +19,22 @@
   }
 
   .space-images {
-    display: flex;
-    flex-wrap: wrap;
-    width: 100%;
-
+    .space-image-style();
+    
     img {
       width: 28%;
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
     }
+    @media @phone {
+      img:not(:first-child) {
```

Comment:
Best code I've seen for setting the display none property. You might want to think about adding comments to code like this. Just so others are aware and you are aware when you look back what it is you were doing. Great Job.